test/Program.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+
+namespace test
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            myclass m = new myclass();
+            //Console.WriteLine("Hello World!");
+            /* Console.WriteLine(sumar(456, 58));
+            Console.WriteLine(m.hi());
+            m.CallMe(15); */
+            m.MayorValor();
+            Console.ReadLine();
+        }
+
+        static int sumar(int num1, int num2)
+        {
+            return num1 + num2;
+        }
+    }
+
+    interface IMyclass
+    {
+        void CallMe(int a);
+        void MayorValor();
+    }
+
+    class myclass : IMyclass

```

Comment:
I let you like an examplo, but name of class  must to start with uppercase (PascalCase)